### PR TITLE
Fix broken links - November 2020

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #21204.

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -11631,6 +11631,10 @@
             "source_path": "docs/standard/serialization/add-element-for-xmlschemaimporterextensions.md",
             "redirect_url": "/dotnet/standard/serialization/add-element-for-schemaimporterextensions"
         },
+        {
+            "source_path": "docs/standard/serialization/basic-serialization-technology-sample.md",
```
Comment (gewarren) [inline .openpublishing.redirection.json:11635]: This serialization sample docs page only gets about 100 page views/month. The link to download the sample is broken and I can't find a replacement, so I deleted the article.

File: docs/core/testing/index.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Keep in mind there are [best practices](unit-testing-best-practices.md) for writ
 
 ### xUnit
 
-[xUnit](https://xunit.net) is a free, open source, community-focused unit testing tool for .NET. Written by the original inventor of NUnit v2, xUnit.net is the latest technology for unit testing .NET apps. xUnit.net works with ReSharper, CodeRush, TestDriven.NET, and [Xamarin](/apps/xamarin). It is a project of the [.NET Foundation](https://dotnetfoundation.org) and operates under their code of conduct.
+[xUnit](https://xunit.net) is a free, open source, community-focused unit testing tool for .NET. Written by the original inventor of NUnit v2, xUnit.net is the latest technology for unit testing .NET apps. xUnit.net works with ReSharper, CodeRush, TestDriven.NET, and [Xamarin](https://dotnet.microsoft.com/apps/xamarin). It is a project of the [.NET Foundation](https://dotnetfoundation.org) and operates under their code of conduct.
```
Comment (gewarren) [inline docs/core/testing/index.md:41]: Is David Coulter's tool trimming these dotnet.microsoft.com links incorrectly?

File: docs/csharp/language-reference/operators/null-forgiving.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ In the preceding example, you don't need to use the null-forgiving operator beca
 
 ## C# language specification
 
-For more information, see [The null-forgiving operator](~/_csharplang/proposals/csharp-8.0/nullable-reference-types-specification.md#the-null-forgiving-operator) section of the [draft of the nullable reference types specification](~/_csharplang/proposals/csharp-8.0/nullable-reference-types-specification.md).
+For more information, see [The null-forgiving operator](~/_csharplang/proposals/csharp-8.0/nullable-reference-types-specification.md#the-null-forgiving-operator) section of the [draft of the nullable reference types specification](~/_csharplang/proposals/csharp-9.0/nullable-reference-types-specification.md).
```
Comment (pkulikov) [inline docs/csharp/language-reference/operators/null-forgiving.md:48]: For this file there is already a PR #21522

**Review by BillWagner — APPROVED** [review]
This LGTM, pending the conflicts and a fresh build.

You can :shipit: once you've addressed the merge conflict.

**Pull request merged**